drafts/develop-a-web-editor-with-react-and-monaco-with-language-server-support.md

```diff
@@ -0,0 +1,334 @@
+---
+title: Develop a Web Editor for your DSL using React and Monaco Editor library with Language Server support
+domain: software-engineering-corner.hashnode.dev
+tags: Web Editor, Monaco, Monaco Language Client, Language Server, LSP
+cover: https://cdn.hashnode.com/res/hashnode/image/upload/v1720940174478/yjy4aJtIT.gif?auto=format
```

Comment:
nitpick:
would it be possible to make this dark mode so it fits better into our blog?